src/Walker/Collector.C

```diff
@@ -113,33 +59,46 @@ Collector::chareOrdPDF( const std::vector< tk::UniPDF >& updf,
   for (const auto& p : tpdf) m_ordtpdf[i++].addPDF( p );
 
   // If all chares on my PE have contributed, send partial sums to host
-  if (m_nopdf == m_nchare) {
+  if (m_nord == m_nchare) {
+
+    // Create Charm++ callback function for reduction
+    CkCallback c1( CkReductionTarget( Distributor, estimateOrd ), m_hostproxy );
```

Comment:
That used to be `c1` and `c2` in a previous attempt of solving this problem which required two reductions starting from the same function to avoid name clashes. Now that is not required, but who cares?